### PR TITLE
feat(frontend): support for no index on transactions page

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

On the transactions page, if the ic transactions store is set to `null`, we assume this is because the token has no index canister and per extension, we want to show a related explanation and placeholder.

# Changes

- New "no index" placeholder
- Use placeholder
- Update skeleton to load while `undefined` instead of `nullish`

# Tests

I just provide a test to check if this or that placeholder is set. We need more tests but, felt like it's quite some work not related to this feature.

# Screenshot

Capture d’écran 2024-11-13 à 17 32 51

### CODE REVIEW

File: src/frontend/src/lib/components/icons/IconSearchClose.svelte [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!-- source: DFINITY foundation -->
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/icons/IconSearchClose.svelte:1]: this is not from Lucide? I had understood that we were using only from Lucide

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks!

File: src/frontend/src/lib/components/icons/IconSearchClose.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/icons/IconSearchClose.svelte:1]: I explicitely asked Artem if I should export from Figma and he said yes. So assumed it was de facto a DFINITY foundation icon. Is that incorrect?

**Comment by peterpeterparker** [comment]
Relates to https://github.com/dfinity/oisy-wallet/issues/2883

**Pull request merged**